recipes-bsp/imx-test/imx-test_git.bb

```diff
@@ -56,8 +56,8 @@ do_compile() {
     oe_runmake V=1 VERBOSE='' \
                CROSS_COMPILE=${TARGET_PREFIX} \
                INC="-I${STAGING_INCDIR} \
+                    -I${STAGING_IMX_INCDIR}" \
```

Comment:
Isn't the order here wrong? The source headers should have a higher priority than the sysroot headers, in which case line 58 is also wrong.

Reply:
You are right, can you fix it?